unify-front-end/supabase/migrations/20260414_seed_learn_modules_metadata.sql

```
@@ -0,0 +1,17 @@
+-- Seed learn_modules with interests and personas mapped from onboarding quiz options.
+-- interests values match LearningInterest type: documents, employment, finance, housing, pr_immigration, healthcare, family_kids, transit
+-- personas values match Persona type: international_student, skilled_worker, refugee
+
+UPDATE learn_modules SET interests = '{documents}', personas = '{}' WHERE sanity_id = '0777d5d9-02fe-4e7d-b644-2e25d1c4579a'; -- Documentation
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)